### PR TITLE
Fixes e2e rollup golden test on windows

### PR DESCRIPTION
**Pull request by gregmagolan**
/cc @alexeagle

### CODE REVIEW

**Comment by bazel-io** [comment]
Can one of the admins verify this patch?

File: internal/e2e/rollup/rollup.spec.js [hunk 1]
```diff
@@ -16,16 +17,24 @@ if (process.env[update_var] && process.env['TEST_SRCDIR']) {
 }
 
 function check(actual, expected) {
-  if (read(actual) !== read(expected)) {
+  const actualContents = read(actual);
+  const expectedContents = read(expected);
+  if (actualContents !== read(expected)) {
```
Comment (alexeagle) [inline internal/e2e/rollup/rollup.spec.js]: `read(expected)` should be `expectedContents`

Comment (gregmagolan) [reply internal/e2e/rollup/rollup.spec.js]: Whoops

Comment (gregmagolan) [reply internal/e2e/rollup/rollup.spec.js]: Fixed

**Pull request merged**